tests/network-setups/besu/scripts/setupNetwork.sh

```diff
@@ -36,46 +38,57 @@ cd ../../../../
 
 
 # Setup EthSigner with multiple signing keys
-bash scripts/setupEthSigner.sh
-cp -r artifacts/defaultKeys/* SampleBesuNetwork/keys/
+bash scripts/setupEthSigner.sh $1
+cp artifacts/network$1/configEthSigner.toml SampleBesuNetwork$1/
 
-# Run the bootNode (Node-1 for us): 
-cd SampleBesuNetwork/Node-1/
-cp ../../artifacts/config1.toml .
+cp -r artifacts/defaultKeys/* SampleBesuNetwork$1/keys/
 
+# Set up the config file for the validator nodes 
+cd SampleBesuNetwork$1/Node-1/
+cp ../../artifacts/network$1/config1.toml .
+sed -i 's,miner-coinbase="0xfe3b557e8fb62b89f4916b721be55ceb828dbd73",miner-coinbase="'"${firstNode}"'",g' config1.toml
 
 # Run for other nodes:
 #bootnodeID=$(curl -X POST --data '{"jsonrpc":"2.0","method":"net_enode","params":[],"id":1}' http://127.0.0.1:8545 | python3 -c "import sys, json; print(json.load(sys.stdin)['result'])")
 
 cd ../Node-2/
-cp ../../artifacts/config2.toml .
+cp ../../artifacts/network$1/config2.toml .
+sed -i 's,miner-coinbase="0xfe3b557e8fb62b89f4916b721be55ceb828dbd73",miner-coinbase="'"${firstNode}"'",g' config2.toml
 
 cd ../Node-3/
-cp ../../artifacts/config3.toml .
+cp ../../artifacts/network$1/config3.toml .
+sed -i 's,miner-coinbase="0xfe3b557e8fb62b89f4916b721be55ceb828dbd73",miner-coinbase="'"${firstNode}"'",g' config3.toml
 
 cd ../Node-4/
-cp ../../artifacts/config4.toml .
+cp ../../artifacts/network$1/config4.toml .
+sed -i 's,miner-coinbase="0xfe3b557e8fb62b89f4916b721be55ceb828dbd73",miner-coinbase="'"${firstNode}"'",g' config4.toml
 
-cd ../../
+cd ../../SampleBesuNetwork$1
 
 # User instructions
 #echo "To complete the setup of the SampleBesuNetwork do the following:" 
 #echo "Open five terminals and run these:"
-echo "Five tmux sessions will be started to run EthSigner and the four validator nodes:"
+echo "Five tmux sessions will be started for Network $1 to run EthSigner and the four validator nodes: EthSigner_session and Node1_session to Node4_session"
 
-tmux new-session -d -s EthSigner_session 'cd  SampleBesuNetwork; ethsigner --chain-id=1337 --downstream-http-port=8590 multikey-signer --directory=keys/'
+#chainId=$(($1 + 1336)) # chainId starts from 1337 for Network 1 and increments by 1 for each network
+#downstreamPort=$((1000*$1 + 7590)) #downstream port i.e., port for Node-1, is 8590 for Network 1 and increments by 1000 for each network
+#tmux new-session -d -s EthSigner_session 'ethsigner --chain-id='${chainId}' --downstream-http-port='${downstreamPort}' multikey-signer --directory=keys/'
```

Comment:
Can you delete commented out lines like these if you don't need them for the future?

Reply:
Yeah, will do that. Thanks for pointing it out.